src/main/java/com/github/manolo8/darkbot/core/utils/EntityList.java

```diff
@@ -128,6 +129,21 @@ private void onEntityCreate(Entity entity) {
         this.eventBroker.sendEvent(new EntityCreateEvent(entity));
     }
 
+    //TODO: add to EntitiesAPI.class
```

Comment:
Agreed, needs to be added to the API.

Maybe also add the same treatment for createFakeNpc, and migrate the current FakeNpc (used for npc locator) to be more in-line with everything else here.

Reply:
Moved to API. PR in description.

Reply:
Also, added not implemented API for fake ship type.